client/src/components/ExpressionResult.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="exp-box">
+    <div class="left-exp-box"></div>
+    <div class="right-exp-box"></div>
+  </div>
+</template>
+
+<style scoped>
+.exp-box {
+  height: 24.875rem;
```

Comment:
```suggestion
  min-height: 24.875rem;
```
最小高さにしたいので`min-height`がよさそうです